vector/src/test/java/im/vector/app/test/fakes/FakePushersService.kt

```diff
@@ -29,17 +29,28 @@ import org.matrix.android.sdk.api.session.pushers.PushersService
 
 class FakePushersService : PushersService by mockk(relaxed = true) {
 
+    fun givenGetPushers(pushers: List<Pusher>) {
+        every { getPushers() } returns pushers
+    }
+
     fun givenPushersLive(pushers: List<Pusher>) {
         every { getPushersLive() } returns liveData { emit(pushers) }
     }
 
-    fun verifyOnlyTogglePusherCalled(pusher: Pusher, enable: Boolean) {
+    fun verifyOnlyGetPushersLiveAndTogglePusherCalled(pusher: Pusher, enable: Boolean) {
         coVerify(ordering = Ordering.ALL) {
             getPushersLive() // verifies only getPushersLive and the following togglePusher was called
             togglePusher(pusher, enable)
         }
     }
 
+    fun verifyOnlyGetPushersAndTogglePusherCalled(pusher: Pusher, enable: Boolean) {
+        coVerify(ordering = Ordering.ALL) {
+            getPushers() // verifies only getPushersLive and the following togglePusher was called
```

Comment:
Comment should be updated I guess by replacing `getPushersLive()` by `getPushers()`. Or comments in these methods could be removed.